examples/handlebars_templates/src/main.rs

```diff
@@ -46,6 +47,27 @@ fn rocket() -> rocket::Rocket {
         .catch(errors![not_found])
 }
 
+// when you want to register handlebars helpers
+#[allow(dead_code)]
+fn rocket_with_hbs_helper() -> rocket::Rocket {
```

Comment:
Don't create a function that's not used. Instead, perhaps you can name this function something like:

```rust
fn install_helper(handlebars: &mut Handlebars) { .. }
```

 and then call it from `rocket()`:

```rust
.attach(Template::custom(|engines| install_helper(&mut engines.handlebars)));
```